WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditOrderAddressForm.swift

```diff
@@ -146,6 +149,15 @@ struct EditOrderAddressForm<ViewModel: AddressFormViewModelProtocol>: View {
                     viewModel.userDidCancelFlow()
                 })
             }
+            ToolbarItem(placement: .automatic) {
+                if isSearchCustomersEnabled {
+                    Button(action: {
+                        viewModel.customerSearchTapped()
+                    }, label: {
+                        Image(systemName: "magnifyingglass")
+                    })
+                }
+            }
```

Comment:
This needs to be checked fully, but I had a look on a 15.5 iPhone 13 mini simulator and there were no layout issues with the flag disabled.

Reply:
I tested this further in different devices, both portrait/landscape, as well as dark mode and no layout issues either.

As a follow-up of my question about trying to add the feature flag check before the `ToolbarItem` itself, rather than before the `Button`, the error I'm getting is: `Closure containing control flow statement cannot be used with result builder 'ToolbarContentBuilder'`. 

From what I understand the problem here is that the toolBar is expecting to get a View and "does not understand" the control flow statement, an alternative could be to use a SwiftUI's `ForEach`, which returns a View, and perform some sort of feature flag check when rendering the `ToolBarItems`. To not add extra complexity, and as doesn't seem to cause any sort of layout issues I'll leave it as it is for the moment 👍

Reply:
Yeah, leave it as is.

[This is another way to do it which looks nice](https://pacugindre.medium.com/swiftui-conditionally-showing-navigationitem-and-toolbaritem-d4d3554d1245)